media/js/firefox/os/devices.js

```diff
@@ -260,27 +259,15 @@ if (typeof window.Mozilla === 'undefined') {
     // GA specific interactions
 
     // track all 'regular' links (non-CTA, non-device)
-    $('.standard-link').on('click', function(e) {
-        var newTab = (this.target === '_blank' || e.metaKey || e.ctrlKey);
-
-        var href = this.href;
-
-        if (newTab) {
-            gaTrack(['_trackEvent', '/os/devices/ Interactions', 'link clicks', href]);
-        } else {
-            e.preventDefault();
-
-            var callback = function() {
-                window.location = href;
-            };
-
-            gaTrack(['_trackEvent', '/os/devices/ Interactions', 'link clicks', href], callback);
-        }
-    });
+    $('.standard-link').attr('data-track', 'true');
```

Comment:
There seems to be a lot of stuff like this that could just be set in the template. I'm not suggesting we change it given the scope of this kind of thing, but it seems like there is a lot of inconsistency on where/how data attributes are set.